src/Directory.Build.props

```diff
@@ -5,6 +5,11 @@
   <!-- This file is imported by all projects at the beginning of the project files -->
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
+  <PropertyGroup>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
Enable Embedded PDBs instead. Much better experience and requires no extra work.